_amenities/klahanie-neighborhood-patrol.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: default 
+title: Klahanie Neighborhood Patrol
+date: 2019-06-10 03:42
+author: Klahanie
+categories: []
+show-title: true
+---
+<div class="float-right col-md-4">
+  <a href="{{site.url}}vacation-house-check.html">Vacation House Check Form</a>
+  <img src="{{site.url}}images/canine-patrol.jpg" class="img-thumbnail">
+  <img src="{{site.url}}images/KNP-2.jpg" class="img-thumbnail">
+  <img src="{{site.url}}images/shoreline-vehicle-2.jpg" class="img-thumbnail">
+</div>
+
+
+<p class="lead">Providing unduplicated levels of neighborhood security</p>
+<em>by Sgt. Steve Lysaght</em>
+
+For the past seven years it has been my pleasure to be the Director of your Klahanie Neighborhood Patrol (KNP).  I am also a Sergeant with the King County Sheriff’s Office.
+
+Each year the KNP program has continued to grow and evolve in providing you with additional police-related services.  With the turnover in residents, I thought it best to re-introduce ourselves to give each of you a better understanding of the enhanced security you receive.

```

Comment:
This isn't the greatest grammar; but, as this is a letter written by an actual person, I'm not going to change the wording.